content/en/tracing/llm_observability/api.md

```diff
@@ -0,0 +1,331 @@
+---
+title: LLM Observability API
+---
+<div class="alert alert-warning">LLM Observability is in public beta, and this API is subject to change. If changes occur, Datadog will provide release notes with any applicable upgrade instructions.</a></div>
+
+The LLM Observability API provides an interface for developers to send LLM-related traces and spans to Datadog. If your application is written in Python, you may prefer to use the [Python SDK][1].
+
+## Spans API
+Use this endpoint to send spans to Datadog. For details on the available kinds of spans, see [Span Kinds][2].
+
+Endpoint
+: `https://api.<DATADOG_SITE>/api/unstable/llm-obs/v1/trace/spans`
+<br />Replace `<DATADOG_SITE>` with the Datadog site you want to send data to (for example, `datadoghq.com` or `us3.datadoghq.com`).
+
+Method
+: `POST`
+
+### Request
+
+#### Headers (required)
+- `DD-API-KEY=<YOUR_DATADOG_API_KEY>`
+- `Content-Type="application/json"`
+
+#### Body data (required)
+
+{{< tabs >}}
+{{% tab "Model" %}}
+| Field | Type | Description                  |
+|-------|------------------------------|------|
+| data [*required*]|  [SpansRequestData](#spansrequestdata) | Entry point into the request body. |
+{{% /tab %}}
+
+{{% tab "Example" %}}
+{{< code-block lang="json" >}}
+{
+  "data": {
+    "type": "span",
+    "attributes": {
+      "ml_app": "weather-bot",
+      "session_id": "1",
+      "tags": [
+        "env:dev",
+        "service:weather-bot",
+        "env:staging",
```

Comment:
Two `env` tags, not sure if that is intentional but just a note~

Reply:
@lievan, I should remove one of these `env` tags, correct?

Reply:
I've removed the `env:dev` tag, but let me know if it was supposed to be there.